atlas-content-modeler/tutorial/graphql/introduction.mdx

```diff
@@ -0,0 +1,40 @@
+---
+slug: /atlas-content-modeler/tutorial/graphql/introduction
+title: Atlas Content Modeler - GraphiQL
+description: How to use GraphiQL
+---
+
+**Note**: GraphiQL is not an ACM specific plugin. It is used in conjunction with ACM.
+
+### Reference
+---
+1. [GraphiQL - NPM](https://www.npmjs.com/package/graphiql)
+
+## GraphiQL
+
+GraphiQL is a plugin that allows you to run queries and has many other features.
+
+### Use GraphiQL with Models
+---
+1. Click on Content Modeler in the WordPress side menu
+
+    ![Atlas Content Modeler sync](../../images/acm-sidebar.png)
```

Comment:
I recommend updating the alt text in the Markdown image markup so that it describes the image. For example:

```suggestion
    ![The Atlas Content Modeler menu item in the WordPress admin sidebar.](../../images/acm-sidebar.png)
```

All of them say, “Atlas Content Modeler sync” at the moment, which isn't helpful for screen reader users or if images fail to load.